### DISCUSSION TITLE
[Feature] Add Map Replacement Feature

### DISCUSSION THREAD

**Discussion by whxciotw** [immich-app/immich#5920] [Feature Request]
### The feature

1, The current use of OpenStreetMap: Is it possible to add a map replacement feature that allows users to switch to common maps of Mainland China (such as Amap or Baidu Map) themselves, and display geographic location names?
2, The current map aggregation granularity is too fine: Can we add the function to aggregate at the city or provincial level?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can configure the map style URLs in the admin settings.

**Reply by charleslcso** [reply]
What's map style URLs mean? 

That means we can use another map?
